pages/give-devices.js

```diff
@@ -121,8 +121,11 @@ export default function GiveDevicesPage() {
                   </p>
                 </div>
               </div>
-              <div className="mt-5 md:mt-0 md:col-span-2">
-                <div className="card bordered">
+              <div className="md:mt-0 md-mt-10 md:col-span-2">
+                <div
+                  className="card bordered"
+                  style={{ border: '2px solid #42B2B8' }}
```

Comment:
Again, I believe Tailwind supports this in its own classes.

Reply:
The #42B2B8 color is not part of the Tailwind palette (it comes from Daisy UI). I tried extending the `tailwind.config.js` following the documentation, but I couldn't make it work. I sligthly changed the style so to use only Tailwind classes.
Right now this branch is behind the main, so I'll close this PR and make a new one addressing this same issue.
Cheers